### PR TITLE
Bug fix/move a few actions from js to cpp

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3683/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3728/

File: arangod/RestHandler/RestAdminExecuteHandler.cpp [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    result.add(StaticStrings::Error, VPackValue(false));
+    result.add(StaticStrings::Code, VPackValue(static_cast<int>(rest::ResponseCode::OK)));
+    result.close();
+
+    generateResult(rest::ResponseCode::OK, result.slice());
+    return RestStatus::DONE;
+  }
+
+  try {
+    LOG_TOPIC(WARN, Logger::FIXME) << "about to execute: '" << Logger::CHARS(body, bodySize) << "'";
```
[the remark this answers is not in the document]
Comment (jsteemann) [reply arangod/RestHandler/RestAdminExecuteHandler.cpp]: This just reuses the existing log topic... I basically moved this from JavaScript to C++ trying to make no functional changes whatsoever.
I agree to having a separate log topic, but I don't want to do it in this PR.
Let's address this together with the other issues we will handle in the context of JavaScript execution/Foxx.

[the remark this answers is not in the document]
Comment (danielhlarkin) [reply arangod/RestHandler/RestAdminExecuteHandler.cpp]: That sounds appropriate to me.

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3732/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3746/

**Pull request merged**